src/chrome/content/rules/MIT.xml

```diff
@@ -361,134 +202,67 @@
 	<target host="orgchart.mit.edu" />
 	<target host="pgp.mit.edu" />
 	<target host="picker.mit.edu" />
-	<!--target host="resources.mit.edu" /-->
 	<target host="school-of-engineering-faculty-search.mit.edu" />
-
 	<target host="scratch.mit.edu" />
 	<target host="api.scratch.mit.edu" />
 	<target host="www.scratch.mit.edu" />
-
 	<target host="scripts.mit.edu" />
 	<target host="www.scripts.mit.edu" />
 	<target host="zyan.scripts.mit.edu" />
-
 	<target host="sdm.mit.edu" />
 	<target host="sipb.mit.edu" />
 	<target host="slice.mit.edu" />
-
 	<target host="shop.sloanreview.mit.edu" />
 	<target host="subscribe.sloanreview.mit.edu" />
-
 	<target host="stellar.mit.edu" />
 	<target host="stuff.mit.edu" />
 	<target host="teachingexcellence.mit.edu" />
 	<target host="techtv.mit.edu" />
 	<target host="tll.mit.edu" />
 	<target host="wayf.mit.edu" />
-	<!--target host="webmail.mit.edu" /-->
```

Comment:
WFM, please readd it and remove it from the comments.

Reply:
Same for `^`, `www` and `web`

Reply:
Lookls like `www` has still some issues, but it can be redirected to `web`. Please have a look at the duplicates in `MIT-mismatches.xml`